help-beta/src/scripts/client/adjust_mac_kbd_tags.ts

```diff
@@ -0,0 +1,72 @@
+// Any changes to this file should be followed by a check for changes
+// needed to make to adjust_mac_kbd_tags of web/src/common.ts.
+
+const keys_map = new Map<string, string>([
+    ["Backspace", "Delete"],
+    ["Enter", "Return"],
+    ["Ctrl", "⌘"],
+    ["Alt", "⌥"],
+]);
```

Comment:
Why can't the same file be included in both places? The duplication will likely be unfortunate eventually.

(E.g., you could move that code out of `common.ts` if you don't want other parts of that module)

Reply:
I think it's also defensible to just have separate implementations as you've done given those limitations.